spotify_test.go

```diff
@@ -104,6 +107,52 @@ func TestNewReleasesRateLimitExceeded(t *testing.T) {
 	}
 }
 
+func TestRateLimitExceededReportsRetryAfter(t *testing.T) {
+	t.Parallel()
+	const retryAfter = 2
+
+	handlers := []http.HandlerFunc{
```

Comment:
I've actually never thought about using a slice over a set of a funcs like this in a test - but this is quite nice!